### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by anupamamnair-tw**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (jagatjeevan) [inline .dockerignore:1]: Need to have all the dot files

Comment (jagatjeevan) [reply .dockerignore:1]: and do check for other files not required

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+COPY package.json .
+RUN npm install
+
+COPY . .
+RUN npm run build
+
+
+
+FROM node:16.13.2-alpine as app
+ENV NODE_ENV=production
```
Comment (jagatjeevan) [inline Dockerfile:15]: This has to be in the build stage and not here

File: .dockerignore (hunk 1 above)
Comment (anupamamnair-tw) [reply .dockerignore:1]: Made changes in the new PR from proper branch:

https://github.com/jagatjeevan/nextjs-setup/pull/3

File: Dockerfile (hunk 2 above)
Comment (anupamamnair-tw) [reply Dockerfile:15]: Made changes in the new PR from proper branch:  https://github.com/jagatjeevan/nextjs-setup/pull/3

**Pull request closed without merging**